.gitlab-ci.yml

```diff
@@ -138,10 +142,12 @@ before_script:
     - DOCVFILES=$(find "$LIB/" -name '*.v' -printf "%p ")
     - DOCLIGHTDIRS="$LIB/theories/Init/ $LIB/theories/Logic/ $LIB/theories/Unicode/ $LIB/theories/Arith/"
     - DOCLIGHTVOFILES=$(find $DOCLIGHTDIRS -name '*.vo' -printf "%p ")
+    # stupid find looks in install/ ...
+    - ACCEPT_ALIEN="ACCEPT_ALIEN_VO=1 ACCEPT_ALIEN_OBJ=1"
 
-    - make doc QUICK=true COQDOC_NOBOOT=true COQTEX="$INSTALLDIR/bin/coq-tex" COQDOC="$INSTALLDIR/bin/coqdoc" VFILES="$DOCVFILES" THEORIESLIGHTVO="$DOCLIGHTVOFILES"
+    - make doc QUICK=true COQDOC_NOBOOT=true COQTEX="$INSTALLDIR/bin/coq-tex" COQDOC="$INSTALLDIR/bin/coqdoc" VFILES="$DOCVFILES" THEORIESLIGHTVO="$DOCLIGHTVOFILES" $ACCEPT_ALIEN
 
-    - make install-doc
+    - make install-doc $ACCEPT_ALIEN
```

Comment:
Wouldn't the proper fix be to not check for alien files on install targets (in the Makefile of Coq)?

Reply:
@SkySkimmer WDYT?

Reply:
Adding the folder gitlab uses for installing things in to `FIND_SKIP_DIRS` is also a possibility.

@letouzey By the way why are there separate ACCEPT_ALIEN_FOO variables?

Reply:
I don't feel like messing with Makefile flow control so I just added it to `FIND_SKIP_DIRS`.